packages/libjson-c/libjson-c.spec

```diff
@@ -0,0 +1,53 @@
+%global jsonc_ver 0.18
+%global jsonc_rel 20240915
+
+Name: %{_cross_os}libjson-c
+Version: %{jsonc_ver}
+Release: %{jsonc_rel}%{?dist}
+Summary: Library for JSON
+License: MIT
+Source0: https://github.com/json-c/json-c/archive/refs/tags/json-c-%{jsonc_ver}-%{jsonc_rel}.tar.gz
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the library for JSON
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n json-c-json-c-%{jsonc_ver}-%{jsonc_rel} -p1
+
+%build
+%{cross_cmake} . \
+  -DBUILD_APPS:BOOL=OFF \
+  -DBUILD_SHARED_LIBS:BOOL=ON \
```

Comment:
This isn't needed, `cross_cmake` sets the flag:

https://github.com/bottlerocket-os/bottlerocket-sdk/blob/8ef6f479f9ea4d2f0f8e0552580cda07282ce706/macros/shared#L218